bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/targetComponent/details/SiderealNameEditor.scala

```diff
@@ -115,6 +115,8 @@ final class SiderealNameEditor(mags: MagnitudeEditor) extends TelescopePosEditor
       }
       if (i.magnitudes.nonEmpty) {
         mags.replaceMagnitudes(i.magnitudes.map(_.toOldModel).asImList)
+      } else {
+        mags.replaceMagnitudes(DefaultImList.create())
```

Comment:
Do you need the `if` in this case?  Would it not be the same if you just always use the first branch?

`mags.replaceMagnitudes(i.magnitudes.map(_.toOldModel).asImList)`